extensions/common/context.cc

```diff
@@ -301,16 +301,9 @@ flatbuffers::DetachedBuffer extractLocalNodeFlatBuffer() {
   return fbb.Release();
 }
 
-namespace {
```

Comment:
@kyessenov I (temporarily) removed this because if I wrap the newly added function in the namespace the call in 381 to `extractPeerMetadataFromUpstreamHostMetadata` (from outside of the ns) gets error. Wasn't sure how to overcome this.